src/desktop/components/react/stitch_components/NavBar.tsx

```diff
@@ -43,7 +43,7 @@ export const NavBar: React.FC<NavBarProps> = ({
       <NavBarContainer id="main-layout-header">
         {showStagingBanner && <StagingBanner />}
         <ReactionNavBar />
-        <FlashBanner />
+        <FlashBannerQueryRenderer />
```

Comment:
When ready this will need to be [added here](https://github.com/artsy/force/blob/master/src/v2/Apps/Components/AppShell.tsx#L59) for our new app shell context (which doesn't use stitch).